unit-test/background/privacy-configuration-tests.js

```diff
@@ -0,0 +1,65 @@
+require('../background/mock-browser-api')
+
+const tdsStorageStub = require('./../helpers/tds.es6')
+const tdsStorage = require('../../shared/js/background/storage/tds.es6')
+
+const Site = require('../../shared/js/background/classes/site.es6')
+const utils = require('../../shared/js/background/utils.es6')
+const browserWrapper = require('../../shared/js/background/wrapper.es6')
+
+const contentScriptUtils = require('../../shared/js/content-scope/utils.js')
+
+const testSets = require('./reference-tests/privacy-configuration/tests.json')
+const configs = {
+    'config1_reference.json': require('./reference-tests/privacy-configuration/config1_reference.json'),
```

Comment:
Can we avoid this hard coding at all?

Reply:
as discussed - I'll try to add some grunt magic. Jonathan makes a good point that if we add more tests and add a new config to tests this hardcoded thing will need to be updated which is annoying